src/components/pokeballSelectorModal.html

```diff
@@ -0,0 +1,46 @@
+<div class="modal fade" id="pokeballSelectorModal" tabindex="-1" role="dialog"
+    aria-labelledby="PokeballSelectorModal" aria-hidden="true">
+    <div class="modal-dialog modal-dialog-scrollable modal-sm" role="document">
+        <div class="modal-content">
+            <div class="modal-header">
+                <h5 class="modal-title" id="pokeballSelectorModalLabel">Pokeball Selection</h5>
+                <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+                    <span aria-hidden="true">&times;</span>
+                </button>
+            </div>
+            <div class="modal-body p-0">
+                <div class="table-warning p-1" data-bind="text: App.game.pokeballs.selectedTitle"></div>
+                <div class="card-body p-0 show table-responsive text-left">
+                    <table class="table table-sm m-0">
+                        <thead>
+                            <tr>
+                                <th width="20%">Ball</th>
+                                <th width="80%">Description</th>
+                            </tr>
+                        </thead>
+                        <tbody>
+                            <tr data-bind="template: { name: 'pokeballSelectionTemplate', data: { 'image': 'None', 'type': -1, 'description': 'Do not catch' }}"></tr>
+                            <!-- ko foreach: App.game.pokeballs.pokeballs -->
+                            <tr data-bind="template: { name: 'pokeballSelectionTemplate', data: $data}"></tr>
```

Comment:
I think we should make the entire row clickable, since the image takes up so little space in comparison.